Makefile

```diff
@@ -115,7 +115,8 @@ test:
 	bash scripts/check-bins-for-jemalloc.sh
 
 bench:
-	LOG_LEVEL=ERROR RUST_BACKTRACE=1 cargo bench --all --no-default-features --features "${ENABLE_FEATURES}" -- --nocapture
+	LOG_LEVEL=ERROR RUST_BACKTRACE=1 cargo bench --all --lib --bins --tests --bench misc --bench channel --no-default-features --features "${ENABLE_FEATURES}" -- --nocapture && \
+	LOG_LEVEL=ERROR RUST_BACKTRACE=1 cargo bench --bench raftstore --bench coprocessor_executors --bench hierarchy --no-default-features --features "${ENABLE_FEATURES}"
```

Comment:
How about other benches like hierarchy and raftstore?

Reply:
I prefer to remove this command. From my own experience make bench is not useful at all, since running all benches takes tooooooo long time. Most of the time I only run a specific bench, using `cargo bench` command directly.

Reply:
It's also a good indication that effectively nobody uses this rule since it is broken :)

Reply:
So I'm gonna remove this target. @brson @breeswish